### PR TITLE
Added AppBox Screenshot

### PR DESCRIPTION
**Pull request by vineetchoudhary**
## Project URL
https://github.com/vineetchoudhary/AppBox-iOSAppsWirelessInstallation

## Category
iOS / macOS

## Description
AppBox is a tool for iOS developers to build and deploy Development, Ad-Hoc and In-house (Enterprise) applications directly to the devices from your Dropbox account.

## Checklist

- [x] Edit [applications.json](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/applications.json) instead of [README.md](https://github.com/serhii-londar/open-source-mac-os-apps/blob/master/README.md).
- [x] Only one project/change is in this pull request
- [x] Screenshots(s) added if any
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

File: applications.json [hunk 1]
```diff
@@ -1006,7 +1006,9 @@
       "repo_url" : "https://github.com/vineetchoudhary/AppBox-iOSAppsWirelessInstallation",
       "title" : "AppBox",
       "screenshots" : [
-
+        "https://github.com/getappbox/Home/raw/master/Images/UploadIPA-Dark.png?raw=true",
```
Comment (serhii-londar) [inline applications.json]: Please remove ?raw=true, it cause redirection and ci mark it as an issue.

Comment (vineetchoudhary) [reply applications.json]: Done. Please recheck.

**Comment by serhii-londar** [comment]
@vineetchoudhary Thank you for your contribution! 👍 Awesome app 🚀

**Comment by vineetchoudhary** [comment]
> @vineetchoudhary Thank you for your contribution! 👍 Awesome app 🚀

Thank you. Glad you like.

**Pull request merged**